locale/it/about/community.md

```diff
@@ -0,0 +1,59 @@
+---
+title: Community Committee
+layout: about.hbs
+---
+
+# Commissione Community
```

Comment:
I would not translate this sentence. I would treat the various WGs, team names as proper names. Community Committee should stay as "Community Committee", just like my name does not become Matthew.

Reply:
I don't totally agree with that, in fact I think it's easier to read and understand if we translate roles; I don't think to roles as of names, I give more importance to what they represent to the reader.

Reply:
It probably is, but it makes it unsearchable on the web.

Reply:
What do you mean? Too much distance between english page and italian one can cause problems?

Reply:
Most of the content and the interaction is in English. The name of the github repos are in English etc. Name of things should stay costant so the barrier between Italian and English is as thin as possible.

Reply:
I agree with what @mcollina said as most of the interaction is anyway in english.

Reply:
I have no strong opinion, just want to point it out that most of the other locales do not use the English version (see fr, ko, uk, etc.)

Reply:
Sounds interesting. Should it be also pointed out to other translators? Is this really an issue leading to search problems on the web?

Reply:
It has _always_ been an issue for me. It is like translating the `HTTPResponse` to `RispostaHTTP` in the docs. Why would you do that? It reduces the chances of possible collaboration and understanding of that.

Reply:
Ok. I disagree, but I do not want this initiative to stop. I’ll recuse myself from further reviews (because no, I cannot LGTM this).

Reply:
Why not `# Community Committee (Commissione Community)` ?

Reply:
That's cool to me, maybe I'd prefer Commissione Community (Community Committee) but it's great in both ways

Reply:
@nodejs/intl we can probably write some guidelines on what shouldn't be translated when we come to a consent.

Reply:
Since I don't have guidelines about this, I'll follow @allevo suggestion and use `# Community Committee (Commissione Community) `, applying this paradigm to all names and committees. We can correct it later on if necessary.